packages/evm/src/providers/wagmi.ts

```diff
@@ -0,0 +1,51 @@
+import { Chain, createClient } from 'viem';
+import { createConfig, http, CreateConnectorFn } from 'wagmi';
+import { injected } from '@wagmi/connectors';
+
+/**
+ * Creates and returns a Wagmi config for the passed in Viem chain.
+ *
+ * @example
+ * React: Use with arctic-1 'viem' Chain and WagmiProvider
+ * ```typescript
+ * import { ARCTIC_1_VIEM_CHAIN, createWagmiConfig } from '@sei-js/evm';
+ *
+ * const queryClient = new QueryClient();
+ *
+ * export const WalletProvider = ({ children }: { children: ReactNode }) => {
+ *   return (
+ *       <WagmiProvider config={createWagmiConfig([ARCTIC_1_VIEM_CHAIN])}>
+ *         <QueryClientProvider client={queryClient}>
+ *           {children}
+ *         </QueryClientProvider>
+ *       </WagmiProvider>
+ *   );
+ * };
+ * ```
+ * @returns Helper function to create a wagmi config given a Viem Chain.
+ * @param chains - An array of Chains from 'viem'.
+ * @param additionalConnectors - A list of additional connectors to add to the Wagmi config for other wallets
+ * @category Wagmi
+ */
+export const createWagmiConfig = (chains: [Chain], additionalConnectors: CreateConnectorFn[] = []) => {
```

Comment:
i think we might need accept some more overrides here, some examples i can think of are:
- pollingInterval
- wallet connect id?

rainbowkit for reference - https://github.com/rainbow-me/rainbowkit/blob/b25db9a1f1f5acc96e28a12e38c2156e4fdbf51d/packages/rainbowkit/src/config/getDefaultConfig.ts#L48